test/sequential/test-net-better-error-messages-port.js

```diff
@@ -10,5 +10,5 @@ c.on('connect', common.mustNotCall());
 c.on('error', common.mustCall(function(e) {
   assert.strictEqual(e.code, 'ECONNREFUSED');
   assert.strictEqual(e.port, common.PORT);
-  assert.strictEqual(e.address, '127.0.0.1');
+  assert.match(e.address, /^127\.0\.0\.1|::1$/);
```

Comment:
This currently checks if the `e.address` starts with `127.0.0.1` or ends with `::1`, let's make sure we are validating whole strings:
```suggestion
  assert.match(e.address, /^(127\.0\.0\.1|::1)$/);
```

Reply:
Of course. Never intended it to be this way :)